fcrepo-clients/src/main/java/edu/unc/lib/dl/event/FilePremisLogger.java

```diff
@@ -47,6 +49,8 @@
     private PID objectPid;
     private Model model;
 
+    private static final PidLockManager lockManager = PidLockManager.getDefaultPidLockManager();
```

Comment:
It may not be necessary to do locking in this client since these changes are happening outside of Fedora in a temp file, so it seems unlikely for there to be conflicts within here.